### ISSUE TITLE
Unknown Error during deployment of webapp

### ISSUE THREAD

**Issue by GoodenJ** [Microsoft/vscode-azureappservice#778]
I'm using webpack to bundle my JS files and then deploy it as an express node.js application. Prior to Friday, Dec 7th, 2018, I was deploying without issue. However, I'm now getting an unknown error causing my deployment to fail. I even tried reverting to my last successful commit in the hopes that I had accidentally done something that could affect deployment but the problem persists.

The following was printed in the Output window:

>5:27:35 PM [webapp name]: Creating zip package...
>5:27:36 PM [webapp name]: Starting deployment...
>5:46:50 PM [webapp name]: Updating submodules.
>5:46:50 PM [webapp name]: Preparing deployment for commit id 'abcdefghij'.
>5:46:50 PM [webapp name]: Generating deployment script.
>5:46:50 PM [webapp name]: Using cached version of deployment script (command: 'azure -y --no-dot-deployment -r "D:\local\Temp\zipdeploy\extracted" -o "D:\home\site\deployments\tools" --basic --sitePath "D:\local\Temp\zipdeploy\extracted"').
>5:46:50 PM [webapp name]: Running deployment command...
>5:46:50 PM [webapp name]: Command: "D:\home\site\deployments\tools\deploy.cmd"
>5:46:51 PM [webapp name]: Handling Basic Web Site deployment.
>5:47:26 PM [webapp name]: **An unknown error has occurred.** Check the diagnostic log for details.
>Deployment to "[webapp name]" completed.

**Comment by nturinski** [comment]
HI @GoodenJ,

Sorry to hear about your deployment issues.  I'm not sure what's changed, but could you try two things for me?

1. Could you check the `Log Files` tree item and see if the deployment log has some more information as to why the deployment failed?
1. Does the deployment fail on a brand new app?

**Comment by GoodenJ** [comment]
Hey @nturinski,

I have since fixed my issue but forgot to update this.

I noticed this error code in the log files: 503.0 - Application pool unavailable.

And yes, the deployment failed on both an old and new application.

Eventually, my app service plan's storage space was maxxed and the error messsage in the console changed. After upgrading the plan to something with more space, the deployment began working again.
